packages/neuron-wallet/src/services/sync-progress.ts

```diff
@@ -27,13 +27,13 @@ export default class SyncProgressService {
       .createQueryBuilder()
       .update(SyncProgress)
       .set({ delete: true })
-      .where({ walletId: Not(In(existWalletIds)) })
+      .where({ walletId: Not(In(existWalletIds)), addressType: SyncAddressType.Default })
```

Comment:
What is `addressType` for in this query

Reply:
In the previous, the multisig-config relates to the wallet. So we can remove other wallets' light client sync except for the current wallet. But now the multisig-config is a global config. So we should filter the wallet and ignore multisig configs. The `addressType` is a query for default `lockArgs` to filter multisig `lockArgs`.

Reply:
Got it